olive/cli/olive_cli.py

```diff
@@ -0,0 +1,56 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import sys
```

Comment:
for this file, do we need the olive prefix in file name?

Reply:
actually it is not needed anymore since the parent module is `olive.cli`. I borrowed the original name from transformers. I can name it to `cli.py` or something else too.

Reply:
changed to launcher.